server/config/query-templates.js

```diff
@@ -89,6 +95,65 @@ module.exports = {
         }
       }
     },
+    phoneOrEmailPeopleAggForImages: {
```

Comment:
I don't think phoneOrEmailPeopleAggForImages or sellerPeopleAggs are being used.  If so, you can delete them.